packages/client/verifiable-credentials/src/verifiableCredentialsSDK/types/nft.ts

```diff
@@ -4,6 +4,7 @@ export interface Nft {
     chain: VCChain;
     contractAddress: string;
     tokenId: string;
+    metadata?: any;
```

Comment:
Can we type this a bit mre specifically? Looking at the code in the NFTProjection type in crossmint-main this is the type we're returning:

```
export interface NFTMetadata {
    image: string;
    collection?: NFTMetadataCollection;
    name?: string;
    description?: string;
    attributes?: NFTMetadataAttribute[];
    animation_url?: string;
    external_url?: string;
    uri?: string;
    properties?: NFTPropertiesAttribute;
    credentialId?: string;
}
```

Reply:
I dont think makes much sense because most of that stuff is specific to our implementation and inner working, this interface should work for any nft metadata